syntax = "proto3";

import "google/protobuf/empty.proto";

// Player version struct
message PlayerVersion {
  uint32 version = 1;
  uint32 sub_version = 2;
}

// Player backend server.
service PlayerBackend {
  // Return player version.
  rpc GetPlayerVersion(google.protobuf.Empty) returns (PlayerVersion) {}
}